src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -562,6 +562,15 @@
        (flatten [public-map random-data])
        random-data))})
 
+;; does this need mock data ??
```

Comment:
I think we're going to tear out mock data at some point, and instead create a proper mock server.  So this is fine.